### ISSUE TITLE
Properties tab to edit inventory stack sizes could be larger

### ISSUE THREAD

**Issue by Tripitakotali** [Goz3rr/SatisfactorySaveEditor#164]
It would be nice if the properties window (once an inventory is opened) could be larger or have it so we can resize the window. Right now in order to edit more than 2 slots I have to click the down arrow on the scroll bar or else it skips from showing slot 0 and 1 to showing slot 3 and 4, skipping slot 2 all together

![PRG0zNN](https://user-images.githubusercontent.com/61675273/75644119-3748bc80-5c0f-11ea-9f85-2b7eddb611df.png)

**Comment by budak7273** [comment]
I have been bothered by this as well. Perhaps it would be possible to change the scrolling amount? I don't have much experience with WPF, and haven't been able to find anything helpful on how to implement this. @virusek20 your input, please?

**Comment by glowtape** [comment]
Yep, this needs fixin'. There's limited utility if you can't see the things that need tweaking.
